frontend/templates/troubleshooting/index.tsx

```diff
@@ -977,6 +979,29 @@ function RelatedProblems({
                         </Text>
                      </Box>
                   </Flex>
+                  {countOfAssociatedProblems && (
+                     <Flex
+                        justifyContent="space-between"
+                        alignItems="center"
+                        w="100%"
+                        h="42px"
+                        padding="12px"
+                        bg="gray.100"
+                        borderTop="1px solid"
+                        borderColor="gray.300"
+                     >
+                        <Text>
+                           {countOfAssociatedProblems === 1
+                              ? '1 Problem'
+                              : countOfAssociatedProblems + ' Problems'}
+                        </Text>
+                        <Link href={deviceGuideUrl} textColor="brand.500">
+                           {countOfAssociatedProblems === 1
+                              ? 'view problem'
+                              : 'view all'}
```

Comment:
```suggestion
                              ? 'View Problem'
                              : 'View All'}
```
These should probably be capitalized, right?

Reply:
[The figma](https://www.figma.com/file/QVLUEpDVoyiMM4MXEGq1L9/iFixit---Problems?type=design&node-id=296-42468&mode=design&t=eC47PEgXAJsC001W-0) has it uncapitalized, but we can certainly capitalize them if you think it looks nicer.

Reply:
Let's follow the Figma, and adjust if needed. 👍

Reply:
We're pretty consistent about using title case with links like this. I'm sorry it's a pain and source of confusion, but copy in Figma is usually the last thing we think about, so treat it with skepticism and ask for clarification where it's inconsistent with the rest of the site or reads badly.

In this case, I'd go with "View all" and "View problem", similar to what we do on guide pages ([example](https://www.ifixit.com/Guide/iPhone+X+Battery+Replacement/103390)) with "Show more…" and "Add a comment'.

Reply:
[55ec567](https://github.com/iFixit/react-commerce/pull/2035/commits/55ec5679bf0add529c08b77a0e8cad431d8af870)